contrib/endpoints/src/api_manager/config.cc

```diff
@@ -113,6 +113,36 @@ MethodInfoImpl *Config::GetOrCreateMethodInfoImpl(const string &name,
   return i->second.get();
 }
 
+bool Config::LoadQuotaRule(ApiManagerEnvInterface *env) {
```

Comment:
rules {
    selector: "apiserving.servicecontrol.testing.v1.QuotaTest.User"
    groups {
      group: "ReadGroupUser"
      cost: 3
    }
  }

will be converted to

quota_metrics: {  # came from rules setting
    name: "serviceruntime.googleapis.com/api/consumer/quota_used_count"
    metric_values: {
	  Labels: {
	      key: "/quota_name"
	      value: "ReadGroupUser"
	    }
	  value: {
	      Int64_value: 3
	    }
      }
  }



metric_rules {
    selector: "apiserving.servicecontrol.testing.v1.QuotaTest.User"
    metric_costs {
      key: "chemisttest.googleapis.com/operation/read_book"
      value: 100
    }
  }

will be converted to

quota_metrics: { # came from metric_rules
    name: "chemisttest.googleapis.com/operation/read_book"
    metric_values: {
	value: {
	    int64_value: 100
        }
  }